src/components/App.tsx

```diff
@@ -87,8 +87,6 @@ export default function App() {
     [reloadCurrentWalletInfo],
   )
 
-  debugger
```

Comment:
Locally I have removed the debugger too. If it doesn't serve any purpose, we should remove it.